BasicRealTime/build.gradle

```diff
@@ -35,6 +35,10 @@ android {
 
 dependencies {
     implementation 'com.amazonaws:ivs-broadcast:1.12.1:stages@aar'
+
+    // The IVS SDK works without OkHttp, but customers typically use OkHttp so let's follow that
```

Comment:
Can we rephrase this to be more customer facing since this is our public repo? Maybe something like 
> The Amazon IVS SDK works without OkHttp, but because many applications include it, we are including it this demo app to mimic real world behavior.

Reply:
Updated to use the exact wording you suggested